frontend/appflowy_flutter/lib/workspace/presentation/notifications/widgets/notification_item.dart

```diff
@@ -57,60 +70,108 @@ class _NotificationItemState extends State<NotificationItem> {
         children: [
           GestureDetector(
             onTap: widget.onAction,
-            child: Opacity(
-              opacity: widget.isRead && !widget.readOnly ? 0.5 : 1,
-              child: DecoratedBox(
-                decoration: BoxDecoration(
-                  color: _isHovering && widget.onAction != null
-                      ? AFThemeExtension.of(context).lightGreyHover
-                      : Colors.transparent,
-                  border: widget.isRead || widget.readOnly
-                      ? null
-                      : Border(
-                          left: BorderSide(
-                            width: 2,
-                            color: Theme.of(context).colorScheme.primary,
+            child: AbsorbPointer(
```

Comment:
Maybe you could make the gesture detector opaque to hit testing?

Reply:
I tried, and although that shohld have worked, no hit test behaviour of the gesture detector blocked the child from receiving key events.